src/assets/scss/app.scss

```diff
@@ -86,7 +86,9 @@ em:lang(ja) {
   left: -1000px;
 }
 .tooltip-inner {
+  border-radius: 2px;
   overflow-wrap: break-word;
+  padding-bottom: 2px;
```

Comment:
The Bootstrap default is 3px, so this is a reduction of 1px.

issa wrote:

> either a px or two needs to be added to the top or taken from the bottom or something between

I tried adding 1px to the top, but even that felt like possibly too much padding. I think it's nice for tooltips to be smaller and to not have too much padding.

I also tried removing 2px from the bottom, but that felt like too much.